geode-wan/src/distributedTest/java/org/apache/geode/internal/cache/wan/wancommand/CreateDestroyGatewaySenderCommandDUnitTest.java

```diff
@@ -182,6 +182,42 @@ public void testCreateDestroyGatewaySender() {
         server3);
   }
 
+  /**
+   * GatewaySender with single dispatcher thread
+   */
+  @Test
+  public void testCreateDestroyGatewaySenderWithSingleDispatcherThread() {
```

Comment:
Cool, I ran this test on develop without your changes and it fails (as you already knew, more for anyone else wondering), it passes with this diff and fails without it.